tweet-api/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Flask==2.2.2
+Flask-Cors==3.0.10
```

Comment:
This doesn't seem to be used, or am I missing it somewhere else?

Reply:
@soulshake

Reply:
It's used [here](https://github.com/soulshake/kuberpoets/pull/5/files#diff-785e186ac3ec10909cdeffef9a4f0f859060388982afa30527cbae5de08cb8daR32), but could try removing it to see what breaks because I don't remember :see_no_evil: